src/components/tx-flow/flows/UpsertRecovery/UpsertRecoveryFlowSettings.tsx

```diff
@@ -132,7 +133,7 @@ export function UpsertRecoveryFlowSettings({
               </Typography>
 
               <Typography variant="body2">
-                You can cancel any recovery attempt when it is not needed or wanted.
+                You can cancel any recovery proposal when it is not needed or wanted during this period.
```

Comment:
As per discussion, this will be left as is because the flow is only seen by the owner and it emphasises that the owner has cancellation rights before the Recoverer.